posts/2020-03-04-Rust-survey-2019.md

```diff
@@ -0,0 +1,167 @@
+
+![Preferred install method](/images/2020-03-RustSurvey/17-Preferred-Install-Method.png)
+
+We still see a very large percentage of users using `rustup` to install Rust, followed by Linux distros.
+
+This year we asked about the upgrade to Rust 2018. 92.8% of our users who responded to the survey are using the 2018 edition of Rust. 85% of users said it took little work do do so.
+
+![Comparative improvement](/images/2020-03-RustSurvey/18-Comparative-Improvement.png)
+
+Users indicated that async I/O improved the most in Rust 2018, no doubt related to the release of [async-await](https://blog.rust-lang.org/2019/11/07/Async-await-stable.html), while GUI development and learning curve improved the least. Clearly we still have much to do to improve Rust's learning curve.
+
+## **Rust workflow**
+
+==== Add Chart Here ====
+
+![Which platform are you developing on]()
+
+51% of Rust users developed on Linux, while 24% developed on Windows, and 23% developed on macOS.
+
+Linux continues to be a powerhouse among Rust developers, holding on to roughly 80% of Rust developers.  Windows usage has grown slightly from 31% last year to 34% this year, its second year in a row of growth.
+
+==== Add Chart Here ====
+
+![Which platforms are you developing for]()
+
+Linux and Windows continue to dominate as targets for Rust applications - 55% of Rust users targeted Linux and 24.5% targeted Windows. Following close behind Windows are macOS and Web Assembly at 22% each.
+
+==== Add Chart Here ====
+
+![What editors do you use]()
+
+Vim and VSCode continue to be the most popular editors (23.6% and 34.9% respectively) by far.
+
+## Feeling welcome
+
+We received a lot of feedback on how we can improve Rust and making it feel more welcome to more people. We can't include all of it here, so here is a summary of some of the feedback that stood out to us.
+
+People are in general asking for more learning material about Rust. In terms of expertise it's mainly beginner and intermediate level material being requested. A lot of these requests also asked for video content specifically.
+
+The common blockers that people mention to participating is that they have social anxiety, and accessibility. One of the common reasons mentioned was that some resources are hard to read for people with dyslexia.
+
+==== Add Chart Here ====
+
+![Are you underrepresented in tech]()
+
+(Need data)
+
+# Rust Non-Users
+
+A big part of a welcoming Rust community is reaching out to non-users as well. As we have in years past, we again asked the reasons why people weren't using Rust. 
+
+For users who had used Rust before but had stopped using it, most had stopped using it within a year of starting.
+
+![How long did you use Rust before you stopped?](/images/2020-03-RustSurvey/3-How-Long-Before-Stopped-Rust.png)
+
+When we asked why someone had stopped using Rust, the most common response was "My company doesn't use Rust."
```

Comment:
I would perhaps follow-up with a sentence like: Of course, this immediately raises the question of *why* their companies don't use Rust. The next most popular answers perhaps provide a clue. There are three major themes that stand out:


- Adoption
- Library support
- IDEs
- Then:
  - Learning curve
  - Interoperability
  - Solved more problems
  - Risky